core/domain/stats_services_test.py

```diff
@@ -466,6 +487,11 @@ def test_record_answers_exceeding_one_shard(self):
             self.assertEqual(
                 len(state_answers.get_submitted_answer_dict_list()), 600)
 
+        sample_answers = stats_services.get_sample_answers(
+            self.EXP_ID, self.exploration.version,
+            self.exploration.init_state_name)
+        self.assertLess(len(sample_answers), 600)
```

Comment:
Ah, I see. Yep, I agree it wasn't clear -- I've removed the whole hanging-on-to-the-existing-tests stuff and replaced it with a new suite of tests.